### DISCUSSION TITLE
Will the stateLayout be part of the Maui toolkit

### DISCUSSION THREAD

**Discussion by developer9969** [CommunityToolkit/Maui#343] [Q&A] [answered]
Will the stateLayout be part of the Maui toolkit? 

Cannot see in here and is not in pull request? May be I am missing something.. 

thanks

**Comment by TheCodeTraveler** [comment, marked as answer]
Hi @developer9969! Yes, `StateLayout` has been approved to be added to the .NET MAUI Community Toolkit: https://github.com/CommunityToolkit/Maui/issues/120

We are currently looking for a community contributor to volunteer to complete its implementation. @pictos Is championing the `StateLayout` proposal. 

If you're interested in leading its implemetnation, let @pictos know, and he'll assign #120 to you and help you get it merged into the toolkit!

**Comment by CliffAgius** [comment]
Closed as answered
